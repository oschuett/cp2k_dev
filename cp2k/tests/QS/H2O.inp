&CP2K
 PROGRAM      Quickstep
 IOLEVEL      10
 FFTLIB       FFTSG
&END

&DFT
 FUNCTIONAL   Pade
&END

&QS
 CUTOFF       200
&END

&SCF
 GUESS        ATOMIC
 MIXING       0.4
 EPS_SCF      1.0E-5
&END

&PRINT low
&END

&KIND H
 BASIS_SET    DZV-GTH-PADE
 POTENTIAL    GTH
&END

&KIND O
 BASIS_SET    DZVP-GTH-PADE
 POTENTIAL    GTH
&END

&CELL
 UNIT         ANGSTROM
 ABC          10.0  10.0  10.0
&END

&COORD
 H   0.000000   -0.757136    0.520545
 O   0.000000    0.000000   -0.065587
 H   0.000000    0.757136    0.520545
&END
