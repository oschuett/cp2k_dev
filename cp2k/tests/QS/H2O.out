 
  **** **** ******  **  PROGRAM STARTED AT              Tue Apr 16 18:04:51 2002
 ***** ** ***  *** **   PROGRAM STARTED ON                                lilith
 **    ****   ******    PROGRAM STARTED BY                                 krack
 ***** **    ** ** **   PROGRAM PROCESS ID                                 71600
  **** **  *******  **  PROGRAM STARTED IN   /gpfs/home/krack/work/cp2k/tests/QS
 
 CP2K| Program name                                                           QS
 CP2K| Input file name                                                   H2O.inp
 CP2K| Project name                                                      project
 CP2K| Default FFT library                                                 FFTSG
 CP2K| Default basis set file                                          BASIS_SET
 CP2K| Default external potential file                                 POTENTIAL
 CP2K| Default library path                                                   ./
 CP2K| Global print level                                                     10
 CP2K| Total number of processors                                              1
 CP2K| This output from processor                                              0
 
 ******************************************************************************
 ******************************************************************************
 **                                                                          **
 **     #####                         ##              ##                     **
 **    ##   ##            ##          ##              ##                     **
 **   ##     ##                       ##            ######                   **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####   **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##  **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######   **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##       **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##       **
 **           ##                                                    ##       **
 **                                                                          **
 **                                                ... make the atoms dance  **
 **                                                                          **
 **                         Version 3.0 (June 2001)                          **
 **                                                                          **
 **     Copyright (C) by MPI fuer Festkoerperforschung, Stuttgart (2000)     **
 **                                                                          **
 ******************************************************************************
 ******************************************************************************

 DFT| Exchange-correlation functional                                      Pade
 DFT| Sets of spin states:                                                    1
 DFT| Cutoffs: density                                              0.10000E-09
 DFT|          gradient                                             0.10000E-07
 DFT| XC mesh (PW density)                                              default

 QS| Method:                                                                GPW
 QS| Number of grid levels:                                                   4
 QS| Density cutoff [a.u.]: 1) grid level                                 100.0
 QS|                        2) grid level                                  50.0
 QS|                        3) grid level                                  25.0
 QS|                        4) grid level                                  12.5
 QS| Grid level progression factor:                                         2.0
 QS| Relative density cutoff [a.u.]:                                       25.0
 QS| Interaction thresholds: eps_pgf_orb:                               1.0E-06
 QS|                         eps_core_charge:                           1.0E-12
 QS|                         eps_rho_gspace:                            1.0E-08
 QS|                         eps_rho_rspace:                            1.0E-08
 QS|                         eps_gvg_rspace:                            1.0E-06
 QS|                         eps_ppl:                                   1.0E-12
 QS|                         eps_ppnl:                                  1.0E-12

 CELL| Volume [ANGSTROM**3]:                                           1000.000
 CELL| Vector a [ANGSTROM]:      10.000     0.000     0.000    |a| =     10.000
 CELL| Vector b [ANGSTROM]:       0.000    10.000     0.000    |b| =     10.000
 CELL| Vector c [ANGSTROM]:       0.000     0.000    10.000    |c| =     10.000
 CELL| Angle (b,c) [degree]:                                             90.000
 CELL| Angle (a,c) [degree]:                                             90.000
 CELL| Angle (a,b) [degree]:                                             90.000
 CELL| Grid size for subcell generation                                       2


 TOTAL NUMBERS AND MAXIMUM NUMBERS

  Total number of

  - Atomic kinds:                       2
  - Atoms:                              3
  - Shell sets:                         4
  - Shells:                             9
  - Primitive Cartesian functions:     13
  - Cartesian basis functions:         18
  - Spherical basis functions:         17

  Maximum angular momentum quantum number of the

  - Orbital basis functions:                        2
  - Local part of the GTH pseudopotential:          2
  - Non-local part of the GTH pseudopotential:      0


 ATOMIC COORDINATES IN ANGSTROM

  Atom  Kind  Element       X           Y           Z       Z(eff)    Mass

     1    1   H     1    0.000000   -0.757136    0.520545      1     1.0080
     2    2   O     8    0.000000    0.000000   -0.065587      6    15.9994
     3    1   H     1    0.000000    0.757136    0.520545      1     1.0080


 SCF PARAMETERS

  density guess: ATOMIC
  max_scf:         50
  nrebuild:         1
  eps_scf:       1.00E-05
  eps_eigval:    1.00E-05
  p_mix:         0.40
  work_syevx:    1.00
  max_diis:         4
  eps_diis:      1.00E-01


 SCF WAVEFUNCTION OPTIMIZATION

  Number of electrons:                             8
  Number of occupied orbitals:                     4
  Number of orbital functions:                    17
  Number of independent orbital functions:        17

  Step  Update method            Time           Convergence        Total energy
  -----------------------------------------------------------------------------
     1  Mixing  0.40             2.82          0.9667206418      -17.0158735666
     2  Mixing  0.40             3.09          0.5764436993      -17.0762807485
     3  Mixing  0.40             3.09          0.3484096145      -17.1113246372
     4  Mixing  0.40             3.04          0.2084373741      -17.1318887213
     5  Mixing  0.40             3.01          0.1248346894      -17.1440378370
     6  Mixing  0.40             3.12          0.0747322102      -17.1512719316
     7  DIIS    0.84E-03         3.03          0.0445939273      -17.1555937820
     8  DIIS    0.29E-04         3.06          0.0000583189      -17.1620532587
     9  DIIS    0.33E-04         3.04          0.0000253037      -17.1620535434
    10  DIIS    0.23E-05         3.09          0.0000058559      -17.1620533715

  *** SCF run converged ***


  Total electronic density (r-space):         -7.9999987106        0.0000012894
  Total core charge density (r-space):         8.0000000658        0.0000000658
  Total charge density (r-space):                                  0.0000013552
  Total charge density (g-space):                                  0.0000013552

  Overlap energy of the core charge distribution:                  0.0000000870
  Self energy of the core charge distribution:                   -43.8328905459
  Core Hamiltonian energy:                                        12.9095647296
  Hartree energy:                                                 17.9090277460
  Exchange-correlation energy:                                    -4.1477553882
  Total energy:                                                  -17.1620533715

 -------------------------------------------------------------------------------
 -                                                                             -
 -                                T I M I N G                                  -
 -                                                                             -
 -------------------------------------------------------------------------------
 SUBROUTINE                        CALLS  CPU TIME T  PERFORMANCE   ELAPSED TIME
 CP2K                                  1     32.23 I       ---            32.88
 Quickstep                             1     32.23 I       ---            32.88
 scf                                   1     32.12 I       ---            32.76
 build_kohn_sham_matrix               10     30.36 I       ---            30.91
 fft_wrap_pw1pw2                     110     24.53 I       ---            24.88
 calculate_rho_elec                   10     11.44 I       ---            11.80
 integrate_v_rspace                   10      6.49 I       ---             6.59
 calculate_xc_potential               10      2.33 I       ---             2.33
 PW_SCATTER                           50      2.16 I   1.9 Mcopy           2.16
 init_grid                             1      1.38 I       ---             1.43
 PW_GATHER                            60      0.93 I   6.1 Mcopy           0.98
 PW_COPY                              90      0.64 I  28.8 Mcopy           0.70
 PW_ZERO                              50      0.56 I  32.9 Mzero           0.64
 HARTREE                              10      0.44 I       ---             0.46
 calculate_rho_core                   10      0.38 I       ---             0.38
 init_cube_info                        4      0.35 I       ---             0.38
 PW_ADD                               30      0.14 I  34.5 Mflops          0.14
 read_qs_env                           1      0.11 I       ---             0.12
 -------------------------------------------------------------------------------


  **** **** ******  **  PROGRAM ENDED AT                Tue Apr 16 18:05:57 2002
 ***** ** ***  *** **   PROGRAM RAN ON                                    lilith
 **    ****   ******    PROGRAM RAN BY                                     krack
 ***** **    ** ** **   PROGRAM PROCESS ID                                 71600
  **** **  *******  **  PROGRAM RAN IN       /gpfs/home/krack/work/cp2k/tests/QS
